fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResourceSwagger.java

```diff
@@ -616,6 +616,10 @@ private GetLoansLoanIdFeeFrequency() {}
             @Schema(example = "0.000000")
             public Double totalRepaymentTransaction;
             @Schema(example = "0.000000")
+            public Double totalInterestRefund;
```

Comment:
`totalUnpaidAccruedDueInterest` and `totalUnpaidAccruedNotDueInterest` are missing